integration/go.mod

```diff
@@ -115,6 +115,7 @@ require (
 	github.com/elastic/gosigar v0.14.2 // indirect
 	github.com/emirpasic/gods v1.18.1 // indirect
 	github.com/ethereum/c-kzg-4844 v0.4.0 // indirect
+	github.com/ethereum/go-ethereum v1.13.10 // indirect
```

Comment:
why is this still used?

Reply:
I just updated the references in the EVM code base, the rest are using regular go-ethereum (e.g. RLP encoding stuff, etc)